### PR TITLE
[9.0] Migrate web_m2x_options to 9.0

### PR DESCRIPTION
**Pull request by zakiuu**

### CODE REVIEW

File: web_m2x_options/__openerp__.py [hunk 1]
```diff
@@ -2,13 +2,12 @@
```
Comment (ThomasBinsfeld) [inline web_m2x_options/__openerp__.py:4]: @zakiuu Fix headers, you can find them in the [Maintainer Tools](https://github.com/OCA/maintainer-tools/tree/master/template/module)

File: web_m2x_options/__openerp__.py [hunk 2]
```diff
@@ -2,13 +2,12 @@
...
+    "version": "9.0.0.1.0",
     "depends": [
         'base',
         'web',
     ],
     'data': ['views/view.xml'],
     "author": "0k.io,Odoo Community Association (OCA)",
-    'installable': False,
-    "active": False,
+    'installable': True,
```
Comment (ThomasBinsfeld) [inline web_m2x_options/__openerp__.py:14]: @zakiuu Remove 'installable': True (True is the default value)

File: web_m2x_options/static/src/js/form.js [hunk 3]
```diff
@@ -1,20 +1,55 @@
 /*global openerp, _, $ */
```
Comment (ThomasBinsfeld) [inline web_m2x_options/static/src/js/form.js:3]: @zakiuu Headers ([Maintainer Tools](https://github.com/OCA/maintainer-tools/tree/master/template/module))

**Comment by ThomasBinsfeld** [comment]
@zakiuu The README.txt already contains relevant information but add the missing titles, you can find them in the [Maintainer Tools](https://github.com/OCA/maintainer-tools/tree/master/template/module). Don't forget to add yourself in the  contributors. :wink:

File: web_m2x_options/__openerp__.py (hunk 2 above)
Comment (pedrobaeza) [reply web_m2x_options/__openerp__.py:14]: Better not to do it, because it's used when migrating branches through versions.

**Comment by njeudy** [comment]
👍  great contribution .. have no time to do it myself :)

Code testing OK

File: web_m2x_options/static/src/js/form.js [hunk 4]
```diff
@@ -336,6 +394,4 @@ openerp.web_m2x_options = function (instance) {
                 return values;
             })
         },
-    });
-};
-
+});
```
Comment (lasley) [inline web_m2x_options/static/src/js/form.js:397]: It would be nice if the `M2ODialog` object was returned in order to allow for inherited modifications

**Comment by lasley** [comment]
Thanks @zakiuu - one comment inline, but more of a nice-to-have vs blocking. 👍

**Comment by sbidoul** [comment]
Some 8.0 merge commits are missing. It needs to be redone...

**Comment by zakiuu** [comment]
Pull request ported to PR #418

**Pull request closed without merging**